common/main/java/com/couchbase/lite/internal/replicator/CBLCookieStore.java

```diff
@@ -0,0 +1,31 @@
+//
```

Comment:
FWIW, this is *so* small and so much a part of the AbstractCBLWebSocket, that I really can see it being defined there.  Not important, just another thought.

Reply:
As I'm moving the cookie utility methods there, it feels cleaner to have its own file.

Reply:
Definitely...